site/content/docs/main/config/api-reference.html

```diff
@@ -5471,6 +5488,23 @@ <h3 id="projectcontour.io/v1alpha1.EnvoySettings">EnvoySettings
 <p>NodePlacement describes node scheduling configuration of Envoy pods.</p>
 </td>
 </tr>
+<tr>
+<td style="white-space:nowrap">
+<code>resources</code>
+<br>
+<em>
+<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#resourcerequirements-v1-core">
```

Comment:
not an issue with this PR but we might need to update some tools since this link doesnt seem to work

Reply:
and is for an old k8s version

Reply:
how to?

Reply:
i saw you had updated it